client/contentrouting.go

```diff
@@ -27,10 +28,28 @@ func (c *ContentRoutingClient) Provide(ctx context.Context, key cid.Cid, announc
 		return nil
 	}
 
-	_, err := c.client.Provide(ctx, key, 24*time.Hour)
+	_, err := c.client.Provide(ctx, []cid.Cid{key}, 24*time.Hour)
 	return err
 }
 
+func (c *ContentRoutingClient) ProvideMany(ctx context.Context, keys []multihash.Multihash) error {
+	keysAsCids := make([]cid.Cid, 0, len(keys))
+	for _, m := range keys {
+		keysAsCids = append(keysAsCids, cid.NewCidV0(m))
+	}
+	_, err := c.client.Provide(ctx, keysAsCids, 24*time.Hour)
+	return err
+}
+
+// Ready is part of the existing `ProvideMany` interface, but can be used more generally to determine if the routing client
```

Comment:
> Ready is part of the existing `ProvideMany` interface

What does this mean? I can't find mention of this anywhere else.

Reply:
https://github.com/ipfs/kubo/blob/9c6094e651bd30a11ee7a501bf9aa7eb5cc34deb/routing/wrapper.go#L35-L38